packages/medusa/src/api/store/orders/query-config.ts

```diff
@@ -40,6 +40,9 @@ export const defaultStoreRetrieveOrderFields = [
   "original_shipping_tax_total",
   "original_shipping_subtotal",
   "original_shipping_total",
+  "credit_lines_total",
+  "credit_lines_subtotal",
+  "credit_lines_tax_total",
```

Comment:
super-nit: I wonder if we should make this singular form to be consistent with the rest of the totals